cmd/vsphere-problem-detector/main.go

```diff
@@ -20,7 +21,7 @@ func main() {
 
 func NewOperatorCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "vsphere-monitoring-operator",
+		Use:   "vsphere-problem-detector",
```

Comment:
was this change intentional?

Reply:
Yes, just so the README, the output binary, and the help output would all match up. I can make a separate PR if you'd like though

Reply:
Did you check if it causes any of metrics emitted by this component to change their names btw?

Reply:
From my testing, the metrics labels didn't change and don't have a prefix. Here's a few of the labels I see returned from `localhost:8443/metrics` where the vsphere problem detector is listening:

```
vsphere_rwx_volumes_total
vsphere_node_check_total
vsphere_datastore_total
```